controllers/user.js

```diff
@@ -288,6 +288,42 @@ exports.confirm_reset_password = (req, res, next) => {
     start();
 };
 
+exports.change_password = (req, res, next) => {
+
+    function start () {
+        if (data instanceof Error) {
+
+            return res.warn(400, {message: data.message});
+        }
+
+        mysql.use('master')
+
+            .query(
+              'UPDATE teacher WHERE teacher_id = (?)  SET password = (?);',
+              [req.session.user.teacher_id,req.params.id],
```

Comment:
Why call the new password variable id? Don't pass anything sensitive using params